drivers/wifi/esp32/Kconfig.esp32

```diff
@@ -58,10 +58,11 @@ config ESP32_WIFI_STA_AUTO_DHCPV4
 
 config ESP32_WIFI_AP_STA_MODE
 	bool "Activates the Station/AP co-existence mode."
+	default y if WIFI_USAGE_MODE_STA_AP
```

Comment:
I guess `CONFIG_ESP32_WIFI_AP_STA_MODE=y` should be replaced to `CONFIG_WIFI_USAGE_MODE_STA_AP=y` in `samples/net/wifi/apsta_mode/socs/esp32_procpu.conf`, right?

Reply:
And perhaps `CONFIG_WIFI_USAGE_MODE_STA=y` enabled by default in `samples/net/wifi/shell/prf.conf`?